app/javascript/article-form/components/Options.jsx

```diff
@@ -92,20 +76,31 @@ export const Options = ({
       </div>
     );
   }
+
   if (schedulingEnabled && !readonlyPublishedAt) {
+    const currentDate = moment().format('YYYY-MM-DD');
     publishedAtField = (
       <div className="crayons-field mb-6">
         <label htmlFor="publishedAt" className="crayons-field__label">
```

Comment:
```suggestion
        <label className="crayons-field__label">
```

we no longer have a `publishedAt` input, and I think we need individual aria-labels anyway